### DISCUSSION TITLE
Allow keys returned from an object to be listed as stable result

### DISCUSSION THREAD

**Discussion by ycmjason** [biomejs/biome#3501] [Ideas]
Thanks for the amazing work! Biome is definitely one of the best tools I've used! Great DX!

Regarding https://biomejs.dev/linter/rules/use-exhaustive-dependencies/#stable-results I suggest we also allow specifying keys that will be stable.

For example, when defining a hook like [useBoolean](https://github.com/juliencrn/usehooks-ts/blob/master/packages/usehooks-ts/src/useBoolean/useBoolean.ts#L31-L50), it would be nice if we can specify something like `stableResults: ['toggle', 'setTrue', 'setFalse', 'setValue']`.

**Comment by siketyan** [comment]
I created #6386 for this.
